README.md

```diff
@@ -1 +1,1038 @@
+
+But first, we will need an environment for deploying all three components, an environment to run this script in.
+
+> If you already have ForgeRock platform components running, you may want to skip the next section and make necessary adjustments to any instructions provided there.
+
+## <a id="chapter-030"></a>Setting up the Environment and Running a Platform Sample with ForgeOps
+
+[Back to the Top](#top)
+
+The easiest way to establish a ForgeRock Identity Platform development environment is downloading and installing the [ForgeRock DevOps and Cloud Deployment](https://github.com/ForgeRock/forgeops) example (ForgeOps), and running it in a [Minikube](https://kubernetes.io/docs/setup/minikube/) instance.
+
+The easiest way to accomplish this task is to follow [DevOps Developer's Guide: Using Minikube](https://backstage.forgerock.com/docs/forgeops/6.5/devops-guide-minikube/#chap-devops-implementation-env) article in ForgeRock documentation. You may, however, want to [Start Here](https://backstage.forgerock.com/docs/forgeops/6.5/start-here/) for getting familiar with the ForgeOps concepts.
+
+Further instructions will assume that the ForgeRock platform software is running in Minikube. In addition, we will assume the file structure that exists in the ForgeOps project at the time of this writing.
+
+As you go through the guide and arrive at:
+
+* [Creating a Namespace](https://backstage.forgerock.com/docs/forgeops/6.5/devops-guide-minikube/#devops-implementation-env-namespace)
+
+    The Guide recommends [creating a namespace](https://backstage.forgerock.com/docs/forgeops/6.5/devops-guide-minikube/#devops-implementation-env-namespace) in your Minikube cluster—for reasons of easier maintenance—so that you wouldn't have to remove obsolete objects by hand.
+
+    > Otherwise, per [official Kubernetes recommendation](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#when-to-use-multiple-namespaces), there should be no need for custom namespaces in a single user development environment that Minikube provides.
+    >
+    > The default namespace will provide the scope for your Kubernetes objects. You will not need to set a specific namespace in your current Minikube context, and you will not have to make changes in your trackable deployment files as described in the [Deploying the Platform](https://backstage.forgerock.com/docs/forgeops/6.5/devops-guide-minikube/#chap-devops-implementation-deploy) chapter of the guide.
+
+    In this writing, we will follow the recommendation. We will assume _`my`_ namespace _was_ created.
+
+    <!--
+    Not sure about this section, as it might introduce unnecessary confusion. However, being aware of the default seems valuable information to consider. At the same time, just ignoring the guide that was recommended to follow may be confusing in itself too.
```

Comment:
I agree. I think it would be better to omit this for now. A subsequent guide for script writing in the context of k8s would be useful, though.

Reply:
Just to make sure: omit the entire forgeops section, correct?

Reply:
yes